src/plotconfig.jl

```diff
@@ -161,12 +179,13 @@ function PlotConfig(T::Val{:designmat})
     cfg = PlotConfig()
     config_kwargs!(
         cfg;
-        layout = (;
-            use_colorbar = true,
-            xlabelFromMapping = nothing,
-            ylabelFromMapping = nothing,
+        layout = (; use_colorbar = true,),
+        axis = (;
+            xlabel = "Conditions",
+            ylabel = "Trials",
+            xticklabelrotation = round(pi / 8, digits = 2),
         ),
-        axis = (; xticklabelrotation = pi / 8),
+        colorbar = (; flipaxis = true, labelrotation = -π / 2, label = "Voltage [µV]"),
```

Comment:
this is wrong, the designmatrix has no units, thus the label is missleading

Reply:
you mean it is not a voltage? What is it then?

Reply:
its a designmatrix, it is whatever you have in your .evts table transformed by the formula

Reply:
but theoretically even topoplot colorbar can show something else except Voltage. So maybe it is fine to keep it as a default version. At least if the default figures in documentation have actually Voltage
If not what to put? " "? "Measure"?

Reply:
I recommend you read up a bit on designmatrices, maybe that clears things up. Maybe read Ehinger & Dimigen 2019 peerJ again?

In any case, the label "voltage" would only be fitting if the predictors would depict voltages. but they dont.

I would just leave it empty